docs/src/config_updates.md

```diff
@@ -24,3 +24,13 @@ is set to `false`.
     auto_update = true
     auto_update_interval_hours = 24
 
+### archive_url
```

Comment:
Given that we will soon switch to using a URL where we append `pages.en.zip`, `pages.de.zip` etc. ourselves, maybe we should prefer something more neutral as the name. How do you feel about "archive_source"?